tests/api/test_backups.py

```diff
@@ -503,6 +503,14 @@ async def test_restore_immediate_errors(
     assert resp.status == 400
     assert "No Home Assistant" in (await resp.json())["message"]
 
+    # mock_partial_backup._locations[None]["path"].unlink()
+    resp = await api_client.post(
+        f"/backups/{mock_partial_backup.slug}/restore/partial",
+        json={"background": True},
```

Comment:
This should be invalid. The client library won't let you make a request that asks supervisor to do a partial restore of nothing at all. I guess the schema needs some fixing 🙈

Reply:
Yeah probably make sense. Thanks for adjusting the test, seems sensible now :+1: .